### DISCUSSION TITLE
Options for previous/next buttons position

### DISCUSSION THREAD

**Discussion by capsia37** [pradyunsg/furo#116] [Ideas]
Hi, I've found that the "prev_next_buttons_location" property is not used to determine the location of the previous/next buttons. It would be nice if that would work after switching from the sphinx_rtd_theme. I've got this working with html overrides but it would be better if it was integrated in the theme. I can provide the code as a PR :)
Thank you!

**Comment by pradyunsg** [comment]
Is there any specific reason you want this?

I've not seen this being very common for documentation themes, and they all have consistently had this at the end of the page.

**Reply by capsia37** [reply]
My main reason is to set the property to "None" since most of the pages in our documentation are not intended to be read as chapters in sequence but single pages. This is the option I've found on the original theme and it would be nice if it was fully compatible with this theme to have a seamless transition. I have sometimes seen documentations with prev/next buttons both on the top and bottom, i think this is useful to go to the previous page without scrolling the full page or searching in the sidebar.

**Reply by aroberge** [reply]
I too miss the option of having the prev/next buttons showing up at the top of the page (as well as at the bottom).

**Reply by pradyunsg** [reply]
@aroberge Could you elaborate on why you want that? I've tried this before and I found it inelegant, especially on short pages.

**Reply by aroberge** [reply]
@pradyunsg I know that you mentioned somewhere else that furo was designed with short pages in mind (or something to that effect).  However, I use it for friendly/friendly-traceback where I have many pages, most of which are long and some of which are extremely long.
You can find the docs here: https://friendly-traceback.github.io/docs/index.html; for an example of a very long page, have a look at the tracebacks for various python versions (see https://friendly-traceback.github.io/docs/tracebacks_en_3.6.html for example).  I've separated the tracebacks into "runtime" and "SyntaxErrors" and to go from one page to the other, it was extremely convenient (when I used an adapted version of read-the-docs theme) to have a prev/next button at the top.  Having it as an **option** would be nice.

**Reply by tuukkamustonen** [reply]
I would also like to remove the previous/next buttons completely (by settings `'prev_next_buttons_location': None`). They bring no value in loosely coupled pages (compared to having a multi-page tutorial or similar).

**Reply by jdumas** [reply]
@pradyunsg is there any update on this topic? I would like to hide the next/prev buttons from my website, but it seems Furo still doesn't have an option for that?
